Makefile

```diff
@@ -3,9 +3,9 @@ DBG         ?= 0
 VERSION     ?= $(shell git describe --always --abbrev=7)
 MUTABLE_TAG ?= latest
 IMAGE        = $(REGISTRY)machine-api-operator
-BUILD_IMAGE ?= registry.ci.openshift.org/openshift/release:golang-1.23
+BUILD_IMAGE ?= registry.ci.openshift.org/openshift/release:golang-1.24
```

Comment:
nit: i'd typically put this in the previous commit, and keep the vendor commit clean

Reply:
Ah yes, nvm I think it is not a big deal, we are not going to drop this commit anyway, it is not a fork, I prefer to avoid force pushing new changes if possible